sidebarLearn.json

```
@@ -1,369 +1,391 @@
 [
+	{
+			"title": "What is meilisearch",
+			"slug": "what_is_meilisearch",
+			"routes": [
+					{
+						"source": "learn/what_is_meilisearch/overview.mdx",
+						"label": "Overview",
+						"slug": "overview"
+					},
+					{
+						"source": "learn/what_is_meilisearch/features.mdx",
+						"label": "Features",
+						"slug": "features"
+					},
+					{
+						"source": "learn/what_is_meilisearch/philosophy.mdx",
+						"label": "Philosophy",
+						"slug": "philosophy"
+					},
+					{
+							"source": "learn/what_is_meilisearch/sdks.mdx",
+							"label": "Official SDKs/libraries",
+							"slug": "sdks"
+					},
+					{
+							"source": "learn/what_is_meilisearch/comparison_to_alternatives.mdx",
+							"label": "Comparison to alternatives",
+							"slug": "comparison_to_alternatives"
+					},
+					{
+							"source": "learn/what_is_meilisearch/telemetry.mdx",
+							"label": "Telemetry",
+							"slug": "telemetry"
+					},
+					{
+							"source": "learn/what_is_meilisearch/search_preview.mdx",
+							"label": "Search preview",
+							"slug": "search_preview"
+					},
+					{
+							"source": "learn/what_is_meilisearch/language.mdx",
+							"label": "Language",
+							"slug": "language"
+					},
+					{
+							"source": "learn/what_is_meilisearch/contact.mdx",
+							"label": "Contact",
+							"slug": "contact"
+					}
+			]
+	},
+	{
+			"title": "Getting started",
+			"slug": "learn/getting_started",
```

```suggestion
			"slug": "getting_started",
```

Based off the other examples, the "learn" part of the slug is added automatically.